www/js/diary/list.js

```diff
@@ -172,16 +172,26 @@ angular.module('emission.main.diary.list',['ui-leaflet',
 
     $scope.$on(Timeline.UPDATE_DONE, function(event, args) {
       console.log("Got timeline update done event with args "+JSON.stringify(args));
-      $scope.$apply(function() {
```

Comment:
in `list.js` line 53, read the day to update from the `$stateParams`, falling back to `moment().startOf('day')` only if there is no parameter specified. `readAndUpdateForDay(moment().startOf('day'));`